From: Ananth Jasty <ajasty@cavium.com>
Date: Wed, 15 Apr 2015 17:59:47 -0700
Subject: [PATCH] Changes required for aarch64 support in boost::config.
Origin: upstream, https://github.com/boostorg/config/commit/8787f6b0718b6b0750869ba52bc65d26db665a05
Bug-Ubuntu: http://bugs.launchpad.net/bugs/1557240
Applied-Upstream: commit, 8787f6b0718b6b0750869ba52bc65d26db665a05
Last-Update: 2016-06-08
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: boost1.58-1.58.0+dfsg/libs/config/checks/architecture/arm.cpp
===================================================================
--- boost1.58-1.58.0+dfsg.orig/libs/config/checks/architecture/arm.cpp
+++ boost1.58-1.58.0+dfsg/libs/config/checks/architecture/arm.cpp
@@ -8,6 +8,8 @@
 
 #if !defined(__arm__) && !defined(__thumb__) && \
     !defined(__TARGET_ARCH_ARM) && !defined(__TARGET_ARCH_THUMB) && \
-    !defined(_ARM) && !defined(_M_ARM)
+    !defined(_ARM) && !defined(_M_ARM) && \
+    !defined(__aarch64__) 
 #error "Not ARM"
 #endif
+
